### PR TITLE
Style cart

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: src/css/responsive/tablet-landscape.styl [hunk 1]
```diff
@@ -47,3 +47,10 @@ $nav-height = 54px
   .grid-item:not(:last-child)
     img.social-icon
       margin-right: $margin-tiny
+
+// ******************************************************
+
+// CART
+
+.icon-remove
+  width: 12px
```
Comment (varffs) [inline src/css/responsive/tablet-landscape.styl:56]: does this size relate to type or something? Could it be calculated from that?

**Review by varffs — APPROVED** [review]
Mobile looks pretty good maybe just the checkout button is a little odd. If the answer to my question is no then just merge freely

File: src/css/responsive/tablet-landscape.styl (hunk 1 above)
Comment (mikeyrayvon) [reply src/css/responsive/tablet-landscape.styl:56]: it doesn't relate to type. i was just eyeing it... 🤷‍♂️

**Comment by mikeyrayvon** [comment]
i see what you mean about the cart button on mobile now. i'll improve that

**Comment by mikeyrayvon** [comment]
ok should be all good now

**Pull request merged**